src/common.ts

```diff
@@ -92,3 +108,25 @@ export class TreeBox<T> {
     return new TreeBox(this);
   }
 }
+
+export async function stat(path: string, test: (info: Deno.FileInfo) => boolean) {
+  try {
+    const info = await Deno.lstat(path);
+    return test(info);
+  } catch (err) {
+    if (err instanceof Deno.errors.NotFound) {
+      return false;
+    } else {
+      throw err;
+    }
+  }
+}
+export async function isFile(path: string) {
```

Comment:
Let's remove these functions. It encourages calling `Deno.lstat` too many times. Instead we should get a `Deno.FileInfo` once and use that.